tests/utilities/test_model_helpers.py

```diff
@@ -92,3 +94,28 @@ def on_fit_start(self):
     trainer = Trainer(fast_dev_run=1)
     trainer.fit(model, train_dataloader=model.train_dataloader())
     assert model.on_fit_start_called
+
+
+def test_get_model_size():
+    model = BoringModel()
+
+    size_bytes = get_model_size(model)
+
+    # The BoringModel has a fully connected layer of size 32x2 with a bias resulting in
+    # 67 weights. Each weight is a float32 -- 4 bytes, therefore we expect a size of
+    # 264.
+    assert size_bytes == 264
```

Comment:
prefer defining a module to test directly here vs importing it. if someone changes the BoringModel changes for any reason, this test will fail

Reply:
The BoringModel shouldn't change :) If it does, lot of test will fail :)